### PR TITLE
View profile

### PR DESCRIPTION
**Pull request by vovando**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
+  def view
```
Comment (dieunb) [inline app/controllers/users_controller.rb]: RESTful

File: app/models/task.rb [hunk 2]
```diff
@@ -1,3 +1,6 @@
 class Task < ActiveRecord::Base
   belongs_to :subject
+  has_many :user_tasks
+
+  scope :list_task, ->subject{where subject_id: subject.id}
```
Comment (dieunb) [inline app/models/task.rb]: add a space after `subject`

File: app/views/users/view.html.erb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  </div>
+</div>
+<div class="row">
+  <% @subjects.each do |subject| %>
+    <div class="row-same-height">
+      <div class="col-xs-3 col-xs-height"><%= subject.name %></div>
+      <div class="col-xs-6 col-xs-height col-top">start date: <span class="color-text">(<%= subject.created_at.strftime "%F" %>)</span></div>
+      <div class="col-xs-3 col-xs-height col-middle">
+        <% subject.user_subjects.each do |us| %>
+          <% if us.subject_id == subject.id && us.status == 0 %>
```
Comment (dieunb) [inline app/views/users/view.html.erb]:
```
+          <% if us.subject_id == subject.id && us.status == 0 %>
+            <%= image_tag "train.png" %>
+          <% end %>
+          <% if us.subject_id == subject.id && us.status == 1 %>
+            <%= image_tag "finish.png" %>
+          <% end %>
```
=>
```
<% if us.subject_id == subject.id %>
  <%= image_tag us.status == 0 ? "train.png" : "finish.png" %>
<% end %>
```

File: app/views/users/view.html.erb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+          <% if us.subject_id == subject.id && us.status == 1 %>
+            <%= image_tag "finish.png" %>
+          <% end %>
+        <% end %>
+      </div>
+      <% Task.list_task(subject).each do |task| %>
+        <div class="col-xs-6 col-xs-height"><%= task.name %></div>
+        <div class="col-xs-6 col-xs-height">
+          <% task.user_tasks.each do |ut| %>
+            <% if ut.status == 0 %>
```
Comment (dieunb) [inline app/views/users/view.html.erb]:
```
+            <% if ut.status == 0 %>
+              <%= image_tag "train.png" %>
+            <% end %>
+            <% if ut.status == 1 %>
+              <%= image_tag "finish.png" %>
+            <% end %>
```
Follow above comment

File: app/models/task.rb (hunk 2 above)
Comment (vovando) [inline app/models/task.rb]: @dieunb https://github.com/framgia/coding-standards/blob/master/vn/rails/standard.md#user-content-scope
Chuẩn convention

Comment (dieunb) [inline app/models/task.rb]: OK

File: config/routes.rb [hunk 5]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'user/view'
```
Comment (dieunb) [inline config/routes.rb]: ???

**Pull request merged**